### PR TITLE
Issue#12 - PR

### PR DESCRIPTION
**Pull request by kameshsampath**
@sabre1041 and @sherl0cks ,

This is first PR for fixing #12 , as discussed earlier I am splitting the fixes as separate PR's for easy merge and better collaboration.

### CODE REVIEW

**Comment by sherl0cks** [comment]
@kameshsampath why are jboss-as-standalone.sh and jboss-as-domain.sh included as templates?

@sabre1041 we've been using a naming convention of prefixing variables with role name. This PR does not follow that convention. Is that a convention we care about / want to keep?

**Comment by sherl0cks** [comment]
@kameshsampath there also doesn't appear to be any changes to the tasks to actually use these templates. am I missing something?

**Comment by kameshsampath** [comment]
@sherl0cks, 
I did make some changes to the script to accept additional configuration parameters from JBOSS_CONF file, hence thought to move it as template within our role.

if required to prefix the vars with role name, i shall do it and push it as part of next commit
 
no you are not missing anything I am pushing task changes as another commit just to stay inline with what we discussed over the issue #12 to allow easier collaboration and multiple commits.

**Comment by kameshsampath** [comment]
@sabre1041 and @sherl0cks , new updated has beed added for fixing #12 

* added additional handlers to enable and restart the jboss eap service
* fixed the variable names with existing standards of prefixing with role_name e.g. jboss_eap_x
* added copying of service files to copy based on `template` module
* added additional init.d scripts for standalone and domain mode startup with additional parameters from `jboss-as.conf`

The "Copy JBoss EAP" task src has been updated with `jboss_eap_artifact_name` instead of `jboss_eap_artifact_url` to allow the local testing of roles with copying the archive from controller machine

please review and merge, let me know if you need further details

**Comment by sherl0cks** [comment]
With a clean check out, I get the following error.

`One or more undefined variables: 'jboss_eap_runtime_conf_file' is undefined`

File: tasks/main.yml [hunk 1]
```diff
@@ -50,7 +51,6 @@
 - name: Set JBoss EAP Service Files Facts
   set_fact:
     jboss_eap_service_conf_file: "{{ jboss_eap_service_conf_dir }}/{{ jboss_eap_user }}.conf"
-    jboss_eap_runtime_conf_file: "{{ jboss_eap_home }}/bin/standalone.conf"
```
Comment (sherl0cks) [inline tasks/main.yml]: Removing this line is creating a problem

Comment (kameshsampath) [inline tasks/main.yml]: the Issue #12 does not touch this line at all, I did try clean builds multiple times on my Vagrant Vm's it went through

File: tasks/main.yml [hunk 2]
```diff
@@ -130,7 +130,7 @@
 
 - name: Copy JBoss EAP
   copy:
-    src: "{{ jboss_eap_artifact_name }}"
+    src: "{{ jboss_eap_artifact_url }}"
```
Comment (sabre1041) [inline tasks/main.yml]: This modification causes issues when attempting to use the `transfer_method: copy-from-controller`

Comment (kameshsampath) [inline tasks/main.yml]: @sabre1041 and @sherl0cks , sorry its my bad - I changed that to test it in local for quicker fixes and forgot to revert it when pushing back . shall i do the change and push it back or its already updated ?

**Comment by sabre1041** [comment]
@kameshsampath I ran into the same issue as @sherl0cks with the `One or more undefined variables: 'jboss_eap_runtime_conf_file' is undefined` error

**Comment by sherl0cks** [comment]
Closed per @kameshsampath request.

**Pull request closed without merging**